gpkit/tests/t_examples.py

```diff
@@ -177,7 +178,7 @@ def test_model_var_access(self, example):
 
     def test_performance_modeling(self, example):
         m = Model(example.M.cost, Loose(example.M), example.M.substitutions)
-
+        NamedVariables.reset_modelnumbers()
```

Comment:
why is this necessary? (may be worth a comment indicating)

Reply:
When developing I encountered an issue where mode1.var != model.var - that has since disappeared. Not sure how this got resolved, either way I will removed the   NamedVariables line.